scripts/missions/sandoria/2_3_1_Journey_to_Bastok.lua

```diff
@@ -78,15 +78,15 @@ mission.sections =
                 end,
 
                 [423] = function(player, csid, option, npc)
-                    if not npcUtil.giveItem(player, { { xi.item.PICKAXE, 5 } }) then
+                    if not npcUtil.giveItem(player, { { xi.item.PICKAXE, 3 } }) then
                         mission:setVar(player, 'Option', 1)
                     else
                         player:setMissionStatus(mission.areaId, 5)
                     end
                 end,
 
                 [425] = function(player, csid, option, npc)
-                    if npcUtil.giveItem(player, { { xi.item.PICKAXE, 5 } }) then
+                    if npcUtil.giveItem(player, { { xi.item.PICKAXE, 3 } }) then
```

Comment:
I think this changes were also in a different PR.

You will need to drop that extra commit you have here.